### ISSUE TITLE
UI Improvement : Hover Effect on Cards

### ISSUE THREAD

**Issue by crocmons** [DevFeed404/DevFeed-1.0#130]
We can add hover effect on cards so that it enhance better UI for users.
![project-42 GSSoc23](https://github.com/DevFeed404/DevFeed-1.0/assets/106610322/4513f535-f145-4df6-a496-9419f459c819)

**Comment by Divyansh3021** [comment]
Can you elaborate ?

**Comment by PatilHarshh** [comment]
i am capable to do this @Divyansh3021 
Description : "when we take cursor to any of this slide  boarder changes and slightly move forward which looks beautifull"

**Comment by crocmons** [comment]
> Can you elaborate ?

yeah, sure.. When the user hovers over the card, you can apply transformations to the card's image, such as scaling, rotating, or blurring. This effect can create a dynamic and engaging interaction, drawing attention to the card and its content. I want to add the hover effect like this way..

[screencast-retink-lime.vercel.app-2023.05.25-14_13_05.webm](https://github.com/DevFeed404/DevFeed-1.0/assets/106610322/f726607b-87fb-4811-9dcc-5df40adb6b92)

**Comment by ayushgupta4002** [comment]
looks good to me @PatilHarshh Just make sure that it sould not have much blur effect!

**Comment by crocmons** [comment]
> looks good to me @crocmons Just make sure that it sould not have much blur effect!

please add the Gssoc23 level for this issue
